src/Microsoft.ML.Data/DataLoadSave/TransformerChain.cs

```diff
@@ -119,26 +119,46 @@ public TransformerChain(params ITransformer[] transformers)
         }
 
         public DataViewSchema GetOutputSchema(DataViewSchema inputSchema)
+        {
+            // Default to only scoring scope.
+            return GetOutputSchema(inputSchema, TransformerScope.Scoring);
+        }
+
+        public DataViewSchema GetOutputSchema(DataViewSchema inputSchema, TransformerScope scope)
```

Comment:
>GetOutputSchema [](http://example.com/codeflow?start=30&length=15)

Does the user can specify all scopes when calling this method? if not, how the user be able to specify all scopes if needed?

Reply:
This question apply to other places we changed too.

Reply:
I am seeing the tests using `TransformerScope.Everything` which do the trick I guess. If this is true, then ignore my comment.

Reply:
Yeah. `TransformerScope.Everything` is how then can do it all.